### ISSUE TITLE
Build failing on macOS

### ISSUE THREAD

**Issue by gedankenexperimenter** [CapeLeidokos/Leidokos-CMake#4]
I'm out of time to try to get this to work right now, so I'll just dump the errors here:
```
[ 36%] Linking CXX static library libmodel01_Kaleidoscope.a
[ 36%] Built target model01_Kaleidoscope
Scanning dependencies of target model01_hid
[ 37%] Building CXX object CMakeFiles/model01_hid.dir/Users/merlin/kcmake/hardware/keyboardio/avr/libraries/hid/src/HID.cpp.obj
[ 39%] Linking CXX static library libmodel01_hid.a
[ 39%] Built target model01_hid
make[2]: *** No rule to make target `CMakeFiles/model01_HID.dir/depend'.  Stop.
make[1]: *** [CMakeFiles/model01_HID.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by noseglasses** [comment]
Sorry that it took me so long to come back to this.

The issue is fixed. It was due to the peculiarities of MacOS' case insensitive file system.

I added automatic testing with travis for MacOS. See the travis configuration for build instructions on MacOS. Basically it is just telling CMake where the Arduino SDK resides. This is, however, only necessary if the SDK is installed in a non-standard location.

https://github.com/CapeLeidokos/Leidokos-CMake/blob/3615777f6d9d75e6d77e0a86b29477f29a5d9f6e/.travis.yml#L57
